src/components/FeatureViews/Search/AccountsCards.js

```diff
@@ -0,0 +1,166 @@
+import { connect } from "react-redux";
+import React, { useState } from "react";
+import BackButton from "../../BackButton";
+import Grid from "@material-ui/core/Grid";
+import { Card, Paper, Typography } from "@material-ui/core";
+import Tabs from "@material-ui/core/Tabs";
+import Tab from "@material-ui/core/Tab";
+import ListView from "@material-ui/icons/ViewList";
+import CardView from "@material-ui/icons/ViewModule";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import TableCell from "@material-ui/core/TableCell";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import CardContent from "@material-ui/core/CardContent";
+import Button from "@material-ui/core/Button";
+import Chip from "@material-ui/core/Chip";
+import CardActions from "@material-ui/core/CardActions";
+import { withRouter } from "react-router-dom";
+import EmailIcon from "@material-ui/icons/EmailOutlined";
+import PhoneIcon from "@material-ui/icons/PhoneOutlined";
+import Hidden from "@material-ui/core/es/Hidden/Hidden";
+
+import { makeStyles } from '@material-ui/styles';
+
+import { ReactComponent as IDIcon } from "../../identifier.svg";
+
+
+import Avatar from "@material-ui/core/Avatar";
+
+
+
+
+function ProfileCard(props) {
+    const [value, setValue] = useState(0);
+    const [userList, setUserList] = useState([])
+    const [viewToggle, setViewToggle] = useState(true);
+
+    const goToRoute = (route) => {
+        props.history.push(props.match.url + route);
+    }
+    const stringToColor = (string) => {
+        let hash = 0;
+        let i;
+
+        /* eslint-disable no-bitwise */
+        for (i = 0; i < string.length; i += 1) {
+            hash = string.charCodeAt(i) + ((hash << 5) - hash);
+        }
+
+        let colour = "#";
+
+        for (i = 0; i < 3; i += 1) {
+            const value = (hash >> (i * 8)) & 0xff;
+            colour += `00${value.toString(16)}`.substr(-2);
+        }
+        /* eslint-enable no-bitwise */
+
+        return colour;
+    }
```

Comment:
There should be an `accountUtils.js` file in Accounts that exports this function, import that instead.